packages/react-components/babel-preset-global-context/src/transformPlugin.ts

```diff
@@ -0,0 +1,259 @@
+import type { NodePath, PluginObj, PluginPass, types as t } from '@babel/core';
+import { types } from '@babel/core';
+import { declare } from '@babel/helper-plugin-utils';
+import hash from '@emotion/hash';
```

Comment:
q: can't we use nodejs [crypto](https://nodejs.org/docs/latest-v14.x/api/crypto.html#crypto_class_hash) for this?

Reply:
to keep the string small, we use murmur hash from emotion. I think `crypto` only has sha256 or md5

Reply:
is that hash length a big concern ?

Reply:
it will be in partners' bundles so  I think smaller the better